crates/forge/tests/data/trace/tests/test_trace.cairo

```diff
@@ -0,0 +1,13 @@
+use snforge_std::{declare, ContractClassTrait, PrintTrait};
+use snforge_std::trace::{get_last_call_trace, DisplayArrayCallEntryPoint};
+
+use trace_info::{ISimpleContractDispatcherTrait, ISimpleContractDispatcher};
+
+#[test]
+fn test_trace_print() {
+    let contract_address = declare('SimpleContract').deploy(@array![]).unwrap();
+
+    ISimpleContractDispatcher { contract_address }.simple_call(10);
+
+    println!("{}", get_last_call_trace());
+}
```

Comment:
Would be nice to have array printing test

Reply:
It is an array printing test, `get_last_call_trace` returns an array

Reply:
@war-in probably meant printing multiple calls from the latest trace

Reply:
I don't see how this would test anything more than this tbh

Reply:
Because if something is broken when printing an array we will catch that

Reply:
But this prints an `Array`, it's just that it has one element. But okay, will change it xD